### DISCUSSION TITLE
How to change colors in "plot_array"?

### DISCUSSION THREAD

**Discussion by wsc1008-cugb** [modflowpy/flopy#1525] [Q&A] [answered]
When I change the color in plot_array(),the result of corresponding color is not generated. The result shows that only the edge color was changed, instead of the facecolor.
![%(KEK @0O9KL336MK938~@G](https://user-images.githubusercontent.com/74029851/187636196-cac06074-19a7-45c6-b7a6-b4ef0862e674.png)

**Comment by wpbonelli** [comment, marked as answer]
Hi @wsc1008-cugb, when plotting arrays, the color at any point is a function of the array's value there. This is achieved with [colormaps](https://matplotlib.org/stable/tutorials/colors/colormaps.html#overview). You can use `cmap` to override the default colormap, which is `viridis`. For example:

```python
mx = flopy.plot.PlotCrossSection(ax=ax, model=swt, link={'row': 0})
conduct1 = mx.plot_array(hk11, masked_values=[hk2, hk3], cmap="plasma", alpha=0.75)
```

You'll probably want a [sequential colormap](https://matplotlib.org/stable/tutorials/colors/colormaps.html#sequential) for a smooth gradient.

**Reply by wsc1008-cugb** [reply]
Thank you a lot, it works!
